.github/workflows/ci.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: '12'
+          xcode-version: '12.0'
```

Comment:
I assume you intend to use the [`setup-xcode`](https://github.com/marketplace/actions/setup-xcode-version) action here? Right now it's part of `setup-node`.

What we probably need is this:

```
- uses: maxim-lobanov/setup-xcode@v1
      with:
        xcode-version: '12.0'
```

I've added it to the existing workflow update PR #186, please confirm if that looks good. Also please confirm that we do need `12.0` here, and why. Interestingly they don't list `12.0` (but `12.0.1`) for `macos-latest` (which is currently macOS 10.15).

See list of available versions here: https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md#xcode